packages/meta-harness/tests/test_evaluator_worktree_lifecycle.py

```
@@ -237,9 +286,178 @@ def fake_materialize(_worktree_dir, _package_dir):
+    _CURRENT_FIXTURE = "# current trusted fixture\n"
+
+    def _make_trusted_package_dir(self, tmp_path: Path) -> Path:
+        package_dir = tmp_path / "package"
+        (package_dir / "scenarios" / "fixtures").mkdir(parents=True)
+        (package_dir / "scenarios" / "fixtures" / "assert-task-state-outcome.py").write_text(
+            self._CURRENT_FIXTURE, encoding="utf-8"
+        )
+        return package_dir
+
+    def _make_stale_worktree(self, tmp_path: Path) -> tuple[Path, Path]:
+        """stale fixture（変更 + 現行に無いファイル）が tracked な git repo を作る。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)